.github/workflows/ci.yml

```diff
@@ -15,6 +15,9 @@ jobs:
       matrix:
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
+    env:
+      - RUSTFLAGS="-D warnings"
```

Comment:
It seems this line is causing the github ci to crash:

> [Invalid workflow file: .github/workflows/ci.yml#L19](https://github.com/DLR-FT/wasm-interpreter/actions/runs/9970661542/workflow)
The workflow is not valid. .github/workflows/ci.yml (Line: 19, Col: 7): A sequence was not expected

Reply:
Good catch! Fixed it.